cockroachcloud/authorization.md

```diff
@@ -1,44 +1,158 @@
+{{site.data.alerts.callout_info}}
+In a future release, this role will be deprecated in favor of more fine-grained roles for separately administrating organization-level user-management functions, cluster functions, and billing functions.
+{{site.data.alerts.end}}
+  
+### Org developer (legacy)
+This role allows the user to read information for all clusters, and to create and manage SQL users on all clusters. It is considered deprecated in favor of the more fine-grained cluster roles below.
+### Cluster developer
+The minimum access role for clusters. This role allows users to view the details of one or more specified clusters, as well as change their IP allowlist configuration.
+### Cluster administrator
+The administrator role for a specific database cluster, including managing SQL users/roles. This role can be granted to an organization user for one or more specific clusters, or for all clusters in the organization.
+### Cluster creator
+This role alows an organization user to create clusters in an organization; the cluster creator is automatically granted the cluster admin role on clusters they create.
 
-Users may be granted two levels of access:
+## Service accounts
 
-- 'Developer' allows users to:
-  - Read cluster information using the CockroachDB Cloud Console and Cloud API
-- 'Administrator' allows users to:
-  - Create and delete organization users
-  - Create, modify, and delete clusters
-  - Create and manage SQL roles on particular clusters
+Service accounts differ from users in that they authenticate with API keys, rather than through the {{ site.data.products.db }} console.
 
-Within an organization, the unit of database functionality is the *CockroachDB cluster*, which corresponds to a networked set of CockroachDB database nodes. All SQL operations and data storage are distributed across the cluster.
+Service accounts currently operate under a unified authorization model with organization users, and can be assigned all of the same [organization roles](#organization-user-roles) as users.
 
-Each cluster has its own set of SQL roles, which allow authenticated users to execute SQL statements using any PostgreSQL-compatible client.
+However, 'legacy service accounts'andmdash;service accounts created prior to April 15 ???andmdash;still may have roles assigned under the legacy model.
+
+See: [Managing service accounts](managing-access.html#manage-service-accounts)
 
 ## Cluster roles for organization users using Cluster SSO
 
 Cluster Single Sign-On (SSO) for {{ site.data.products.db }} allows authorized organization users to directly access clusters within the organization via [`ccloud`](ccloud-get-started.html), the {{ site.data.products.db }} command line interface.
 
 However, because organization roles and roles on any given SQL cluster are logically separate, a corresponding SQL role must be created for each SSO organization user, on each particular cluster.
```

Comment:
```suggestion
However, because organization users and cluster SQL users are logically separate, a corresponding SQL user must be created for each SSO organization user, on each particular cluster.
```